### ISSUE TITLE
When parsing an invalid template, fatal error is throw.

### ISSUE THREAD

**Issue by laurentmuller** [friendsoftwig/twigcs#129]
When parsing an invalid template like follow:

```twig
<p class="ml-4 mb-0">{{ titleIcon } }</p>
```
An fatal error is throw with the following trace.
```php
Fatal error: Uncaught Error: Class 'FriendsOfTwig\Twigcs\TwigPort\Error' not found in C:\Users\Bibi\AppData\Roaming\Composer\vendor\friendsoftwig\twigcs\src\TwigPort\SyntaxError.php:12
Stack trace:
#0 C:\Users\Bibi\AppData\Roaming\Composer\vendor\composer\ClassLoader.php(444): include()
#1 C:\Users\Bibi\AppData\Roaming\Composer\vendor\composer\ClassLoader.php(322): Composer\Autoload\includeFile('C:\\Users\\Bibi\\A...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('FriendsOfTwig\\T...')
#3 C:\Users\Bibi\AppData\Roaming\Composer\vendor\friendsoftwig\twigcs\src\TwigPort\TwigLexer.php(360): spl_autoload_call('FriendsOfTwig\\T...')
#4 C:\Users\Bibi\AppData\Roaming\Composer\vendor\friendsoftwig\twigcs\src\Lexer.php(44): FriendsOfTwig\Twigcs\TwigPort\TwigLexer->lexExpression()
#5 C:\Users\Bibi\AppData\Roaming\Composer\vendor\friendsoftwig\twigcs\src\Lexer.php(75): FriendsOfTwig\Twigcs\Lexer->lexExpression()
#6 C:\Users\Bibi\AppData\Roaming\Composer\vendor\friendsoftwig\twigcs\src\TwigPort\TwigLexer.php(210):  in C:\Users\Bibi\AppData\Roaming\Composer\vendor\friendsoftwig\twigcs\src\TwigPort\SyntaxError.php on line 12
```

This is because the `SyntaxError` class is extending the `Error` instead of  `\Error`  (global namespace).

The class definition:
```php
class SyntaxError extends Error
```
Must be changed with:
```php
class SyntaxError extends \Error
```
The same problem is detected for the `ScopedExpression` class in line 154.
```php
if (null === $previousHead) {
    throw new RuntimeException('Cannot rollback : no parent scope.');
}
```
Must be changed with:
```php
if (null === $previousHead) {
    throw new \RuntimeException('Cannot rollback : no parent scope.');
}
```

**Comment by laurentmuller** [comment]
I was wrong for the `Error` class. See #124. The `SyntaxError `class must be derived from the Twig\Error. So we have to implements this class.

**Comment by OwlyCode** [comment]
This was fixed by #130
